packages/@aws-cdk/core/test/fs/fs-ignore.test.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+import { nodeunitShim, Test } from 'nodeunit-shim';
+import { IgnoreStrategy } from '../../lib/fs';
+
+nodeunitShim({
```

Comment:
Can I ask you to rewrite these to Jest tests and to use `expect(...).toEqual(...)` ? It'll generate better error messages.

In fact, I think I'd like to see tests of the form:

```ts

describe('globIgnorePattern', () => {

  test('excludes nothing by default', () => {
      const ignore = IgnoreStrategy.glob([]);
      expect(filterFiles(ignore, [
        'some/file/path',
      ]).toEqual([
        'some/file/path',
      ]);
  });

});

function filterFiles(strat: IgnoreStrategy, files: string[]) {
  return files.filter(file => !strat.ignores(file));
}
```

Feels like this would produce understandable outputs if the tests fail for some reason.

Reply:
Sounds good to me. See b47b40402ed7b7573489c4cef7a7900023dbbbd3